staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -291,83 +291,170 @@ func init() {
 	runtime.Must(utilfeature.DefaultMutableFeatureGate.AddVersioned(defaultVersionedKubernetesFeatureGates))
 }
 
+func GetDefaultVersionedKubernetesFeatureGatesForTest() map[featuregate.Feature]featuregate.VersionedSpecs {
```

Comment:
cc @liggitt, this is specifically meant for the history test script, we shouldn't need to commit this right?

Reply:
oops, my bad :) sorry